libraries/SD_MMC/src/SD_MMC.cpp

```diff
@@ -186,6 +246,30 @@ bool SDMMCFS::begin(const char *mountpoint, bool mode1bit, bool format_if_mount_
   }
   _mode1bit = mode1bit;
 
+#ifdef SOC_SDMMC_IO_POWER_EXTERNAL
+    sd_pwr_ctrl_ldo_config_t ldo_config = {
+        .ldo_chan_id = _power_channel,
```

Comment:
what if powered externally?

Reply:
Done using a check if _power_channel is specified.
If not, message is printed to use external power for SDMMC.